spec/models/widget/card_spec.rb

```diff
@@ -17,6 +17,38 @@
     it "is not valid without a title" do
       expect(build(:widget_card, title: "")).not_to be_valid
     end
+
+    context "regular cards" do
+      it "link_url field is required" do
```

Comment:
```suggestion
      it "is not valid without a link_url" do
```

For consistency with the tests preceding this one as well as othet validation tests in other models.